### DISCUSSION TITLE
Wordnet in Spacy

### DISCUSSION THREAD

**Discussion by ronyfsw** [explosion/spaCy#10367] [Help: Model Advice] [answered]
I'm trying to identify synonyms using this standard script from the [Spacy API]([url](https://spacy.io/universe/project/spacy-wordnet)): 
```
import spacy
#from spacy_wordnet.wordnet_annotator import WordnetAnnotator
# Load an spacy model (supported models are "es" and "en")
nlp = spacy.load('en_core_web_sm')
# Spacy 3.x
nlp.add_pipe("spacy_wordnet", after='tagger', config={'lang': nlp.lang})
# Spacy 2.x
# nlp.add_pipe(WordnetAnnotator(nlp.lang), after='tagger')
token = nlp('prices')[0]`
```
I replaced 'en' by the model's full name ('en_core_web_sm') as instructed for 3.0. 

The response is: 
`ValueError: [E002] Can't find factory for 'spacy_wordnet' for language English (en). This usually happens when spaCy calls `nlp.create_pipe` with a custom component name that's not registered on the current language class. If you're using a Transformer, make sure to install 'spacy-transformers'. If you're using a custom component, make sure you've added the decorator `@Language.component` (for function components) or `@Language.factory` (for class components).'

Is the Wordnet bridge available for this or other English model? 

`

**Comment by polm** [comment, marked as answer]
spacy-wordnet is a third party package, like basically everything in the Universe. You need to install it separately. And if you have trouble with it you should ask at [their repo](https://github.com/recognai/spacy-wordnet).

**Reply by ronyfsw** [reply]
Thanks, will do
